### PR TITLE
Assignment 1 - Benjamin Knoll

### PR DESCRIPTION
**Pull request by BenKnoll3**
Azure deployment - https://blue-mud-01e19741e.3.azurestaticapps.net/

Wordle Ideas:
-Scramble feature that allows you to randomly place available letters into the guess prompt.
-A point system that awards points at completion based on guess count and time to complete.
-A leaderboard that shows everyones score(possibly a random name generator to avoid trolls).

Paired w/ @grahamna in the future.

### CODE REVIEW

**Review by Nolan-Posey — COMMENTED** [review]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

Great job guys! The only thing I would recommend is that you play around with some of the different styles for v-card and v-btn just to experiment with and see what else there is to do with them. 

-Nolan Posey

File: wordle-webapp/src/views/game.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
```
Comment (mmwoodfo) [inline wordle-webapp/src/views/game.vue:1]: Your file should be in PascalCase `Game.vue`

File: wordle-webapp/src/views/game.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <div class="game">
```
Comment (mmwoodfo) [inline wordle-webapp/src/views/game.vue:2]: You don't need this div

File: wordle-webapp/src/views/game.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          <v-spacer />
+          <v-btn>
+            <RouterLink to="/">Return Home</RouterLink>
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </div>
+  </template>
+  
+  <style>
```
Comment (mmwoodfo) [inline wordle-webapp/src/views/game.vue:22]: I don't think this is doing anything.

File: wordle-webapp/src/views/game.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  
+        <v-card-text> 
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ultrices sapien orci, nec varius massa tincidunt tincidunt. Suspendisse ultrices commodo ante vitae ullamcorper. Sed sed justo auctor, sodales lectus eu, volutpat odio.  
+          <v-icon icon="mdi:mdi-abacus" />
+        </v-card-text>
+  
+        <v-card-actions>
+          <v-spacer />
+          <v-btn>
+            <RouterLink to="/">Return Home</RouterLink>
```
Comment (mmwoodfo) [inline wordle-webapp/src/views/game.vue:15]:
```suggestion
            Return Home
```

File: wordle-webapp/src/views/game.vue [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        <v-card-subtitle> (CLASS) </v-card-subtitle>
+  
+        <v-card-text> 
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ultrices sapien orci, nec varius massa tincidunt tincidunt. Suspendisse ultrices commodo ante vitae ullamcorper. Sed sed justo auctor, sodales lectus eu, volutpat odio.  
+          <v-icon icon="mdi:mdi-abacus" />
+        </v-card-text>
+  
+        <v-card-actions>
+          <v-spacer />
+          <v-btn>
```
Comment (mmwoodfo) [inline wordle-webapp/src/views/game.vue:14]:
```suggestion
          <v-btn to="/">
```

**Pull request closed without merging**